script/input/1/README.md

```diff
@@ -0,0 +1,25 @@
+# Network 1 (Ethereum Mainnet) Deployment Configuration
+
+This directory contains the configuration files for deploying DSPC on Ethereum Mainnet.
+
+## Files
+
+### dspc-deploy.json
+Configuration for deploying DSPC and DSPCMom contracts:
+- `conv`: Address of the converter contract that handles rate conversions between basis points and ray format
+
+## Usage
+
+1. Update `dspc-deploy.json` with the correct `conv` address
+2. Run the deployment script:
+```bash
+forge script script/DSPCDeploy.s.sol:DSPCDeployScript \
+    --rpc-url $ETH_RPC_URL \
+    --broadcast
+```
```

Comment:
Following up on the `.gitignore` suggestion, it should be something like:
```suggestion
1. Copy `template-dspc-deploy.json` into a new file (i.e.: `dspc-deploy.json`)
2. Edit the new file with the correct `conv` address
3. Run the deployment script:
```bash
FOUNDRY_SCRIPT_CONFIG=dspc-deploy forge script script/DSPCDeploy.s.sol:DSPCDeployScript \
    --rpc-url $ETH_RPC_URL \
    --broadcast
```
```